tests/unit/test_import_package.py

```diff
@@ -153,6 +172,65 @@
     "MyRootPackage/D/E/": None,
 }
 
+test_documents_with_dependency_conflict = {
+    "MyRootPackage/package.json": {
+        "name": "MyRootPackage",
+        "type": "CORE:Package",
+        "_meta_": {
+            "type": "CORE:Meta",
+            "version": "0.0.1",
+            "dependencies": [
+                {
+                    "alias": "CORE",
+                    "address": "system/SIMOS",
+                    "version": "0.0.1",
+                    "protocol": "sys",
+                },
+                {
+                    "alias": "SINTEF",
+                    "address": "marine-modells.sintef.com/Signals",
```

Comment:
is there any authentication required to get data from this address? Or is this just an example?

Reply:
just an example